site/src/helpers/util.tsx

```diff
@@ -115,17 +115,17 @@ function transformCourseGQL(data: CourseGQLResponse) {
   return course;
 }
 
-// TODO: should move transformations to backend
+// TODO: should move transformations to backend? check performance
```

Comment:
prefer jsdoc comments?
```suggestion
/** @todo should move transformations to backend? check performance */
```

Reply:
sure